tests/integration/__tests__/patterns-dirs.js

```diff
@@ -151,27 +151,47 @@ if (path.sep === "/") {
     testPatterns("", ["test-b\\?"], { stdout: "test-b\\?/test.js\n" });
     testPatterns("", ["test-b*/*"], { stdout: "test-b\\?/test.js\n" });
   });
+}
 
-  describe("Ignore symlinks", () => {
-    beforeAll(() => {
-      fs.mkdirSync(path.resolve(base, "test-a"));
-      fs.writeFileSync(path.resolve(base, "test-a", "one.js"), "x");
-      fs.mkdirSync(path.resolve(base, "test-b"));
-      fs.writeFileSync(path.resolve(base, "test-b", "two.js"), "x");
-      fs.symlinkSync("../test-b", path.resolve(base, "test-b", "symlink"));
-    });
-
-    afterAll(() => {
-      fs.unlinkSync(path.resolve(base, "test-a", "one.js"));
-      fs.rmdirSync(path.resolve(base, "test-a"));
-      fs.unlinkSync(path.resolve(base, "test-b", "two.js"));
-      fs.unlinkSync(path.resolve(base, "test-b", "symlink"));
-      fs.rmdirSync(path.resolve(base, "test-b"));
-    });
-
-    testPatterns("", ["test-b"], { stdout: "test-b/two.js\n" });
+describe("Ignore symlinks", () => {
```

Comment:
@fisker We need to skip this test on Windows since Windows doesn’t support symlinks by default, right? That’s why I had put this in the `if (path.sep === "/")` block (but we could also check `process.platform !== "win32"` or something).

Reply:
It seems that Windows users can create symlinks [when Developer Mode is enabled](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/).

Maybe we can safely assume that anyone running the Prettier test suite has that enabled?

Reply:
We can do a feature test then. Like

```js
function isSymbolicLinkSupported() {
  if (path.sep !== "/") {
    return true;
  }

  try {
    fs.symlinkSync(file);
    fs.statSync(file);
    return fs.isSymbolicLink();
  } catch {}

  return false;
}
```

Reply:
Added a feature test.